crates/aiken-lang/src/parser.rs

```diff
@@ -13,10 +13,11 @@ mod utils;
 use crate::{ast, line_numbers::LineNumbers};
 pub use annotation::parser as annotation;
 use chumsky::prelude::*;
-pub use definition::parser as definition;
+pub use definition::{import::parser as import, parser as definition};
 use error::ParseError;
 pub use expr::parser as expression;
 use extra::ModuleExtra;
+use indexmap::IndexMap;
```

Comment:
Note: the `IndexMap` is crucial here to ensure that we then iterate on imports in the same order. Using an `HashMap` would as a side-effect sort the imports alphabetically without properly changing their spans and the comments attached to them if any.